rotkehlchen/tests/api/test_ens.py

```diff
@@ -68,6 +69,9 @@ def test_reverse_ens(rotkehlchen_api_server):
     timestamps_after_request = _get_timestamps(db, addrs_1)
     assert timestamps_before_request == timestamps_after_request
 
+    # Going to check that after request with ignore_cache ens_mappings will be updated
+    db_changes_before = db_conn.total_changes
+
```

Comment:
nitpick: Unnecessary newline

```suggestion
    # Going to check that after request with ignore_cache ens_mappings will be updated
    db_changes_before = db_conn.total_changes

```